src/main/java/juja/sqlcmd/Application.java

```diff
@@ -1,7 +1,218 @@
+                return;
+            }
+
+            StringBuilder stringBuilder = new StringBuilder();
+            int columnsCount = getColumnsCountFrom(resultSet);
+            while (resultSet.next()) {
+                String columnsSeparator = " |";
+                for (int i = 1; i <= columnsCount; i++) {
+                    stringBuilder.append(resultSet.getString(i));
+                    stringBuilder.append(columnsSeparator);
+                }
+                truncate(stringBuilder, columnsSeparator.length());
+                stringBuilder.append(LINE_SEPARATOR);
+            }
+            System.out.println(stringBuilder);
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private void truncate(StringBuilder stringBuilder, int charsCount) {
+        stringBuilder.delete(stringBuilder.length() - charsCount, stringBuilder.length());
+    }
+
+    private void insertIntoUser(String tableName, String name, String password) {
+        String sqlQuery = String.format("INSERT INTO " + tableNameFormatted(tableName) + "(name, password) VALUES('%s','%s') ", name, password);
+        printStatusMsg("Inserting into", tableName);
+        int affectedRows = executeQuery(sqlQuery);
+        printMsgAffectedRows(affectedRows);
+    }
+
+    private void printMsgAffectedRows(int affectedRows) {
+        if (affectedRows < 0) {
+            System.out.println("The query ended unsuccessfully, 0 rows affected");
+            System.out.println();
+        } else {
+            System.out.println("The query ended successfully, " + affectedRows + " row(s) affected");
+            System.out.println();
+        }
+    }
+
+    private int executeQuery(String sqlQuery) {
+        try (PreparedStatement statement = connection.prepareStatement(sqlQuery)) {
+            return statement.executeUpdate();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return -1;
+    }
+
+
+    private int getColumnsCountFrom(ResultSet resultSet) {
+        try {
+            ResultSetMetaData rsmd = resultSet.getMetaData();
+            return rsmd.getColumnCount();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return -1;
+    }
+
+    private void createTableUser(String tableName) {
+        String sqlQuery = "CREATE TABLE IF NOT EXISTS " + tableNameFormatted(tableName) +
+                "(" +
+                "id SERIAL PRIMARY KEY," +
+                "name text," +
+                "password text" +
+                ")";
+        printStatusMsg("Creating", tableName);
+        int affectedRows = executeQuery(sqlQuery);
+        printMsgAffectedRows(affectedRows);
+    }
+
+    private void printTableList() {
+        try {
+            printStatusMsg("Printing", "list");
+            DatabaseMetaData md = connection.getMetaData();
+            ResultSet resultSet = md.getTables(null, "public", "%", new String[]{"TABLE"});
```

Comment:
ResultSet is a resourse and needs to be closed (http://bit.ly/2zHWGEE)

Reply:
Will be fixed next PR